kafka-ui-react-app/src/components/Topics/List/__tests__/ListItem.spec.tsx

```diff
@@ -28,34 +28,42 @@ describe('ListItem', () => {
   );
 
   it('triggers the deleting messages when clicked on the delete messages button', () => {
-    const component = shallow(setupComponent());
-    component.find('DropdownItem').at(0).simulate('click');
-    expect(mockDeleteMessages).toBeCalledTimes(1);
-    expect(mockDeleteMessages).toBeCalledWith(
-      clusterName,
-      internalTopicPayload.name
-    );
+    if (!internalTopicPayload.internal) {
```

Comment:
This condition will never pass since `internalTopicPayload.internal` is always `true`. To keep these tests functional you should setup component using `externalTopicPayload`:
```typescript
const component = shallow(setupComponent({ topic: externalTopicPayload }));
```

Reply:
done